packages/cdktf-cli/bin/cmds/ui/models/terraform.ts

```diff
@@ -49,6 +49,14 @@ export interface TerraformOutput {
   value: string | Record<string, unknown> | Array<any>;
 }
 
+export function isTerraformOutput(output: any): output is TerraformOutput {
+  return (
+    typeof output === "object" &&
+    typeof output.sensitive === "boolean" &&
+    typeof output.type === "string"
```

Comment:
Those are always set, right?

Reply:
Hmm now that you mention it, I trusted on the type system, but let me verify